resolwe/test/testcases/process.py

```diff
@@ -710,6 +710,31 @@ def assertFileExists(self, obj, field_path):  # pylint: disable=invalid-name
         if not os.path.isfile(output):
             self.fail(msg="File {} does not exist.".format(field_path))
 
+    def assertFilesExist(self, obj, field_path, filenames=None):  # pylint: disable=invalid-name
+        """Ensure files in the given object's field exists.
+
+        :param obj: object that includes list of files for which to check
+            existance
+        :type obj: ~resolwe.flow.models.Data
+
+        :param str field_path: path to
+            :class:`~resolwe.flow.models.Data` object's field with the
+            file name/path
+
+        :param str filenames: Expected names of the files.
+        """
+        field = dict_dot(obj.output, field_path)
+
+        output_files = []
+        for item in field:
+            output_file = os.path.join(settings.FLOW_EXECUTOR['DATA_DIR'], str(obj.pk), item['file'])
+            output_files.append(output_file)
+            if not os.path.isfile(output_file):
+                self.fail(msg="File(s) in output field {} don't exist.".format(field_path))
+
+        if filenames:
```

Comment:
This is an unnecessary thing as in our model it only matters that output field references a file and that the file exists on the disk, the naming doesn't matter. 

But if you want to keep it, make sure that `assertFilesExist` and `assertFileExists` are in sync and both support the same options.

Reply:
This comes really handy in the following case: You expect multiple files to be created, but a different set of them, depending on some input. That way, you can check that correct files have been made, but do not need to check their sha256

Reply:
I agree. But my point was that platform would (or may) work even if filenames change, but this will fail in such cases. So I am not in favour of exposing such anti-patterns, as they give the feeling that this is intended and can cause a lot of problems when get widely used.